docs/user-guide/building-a-resource.md

```diff
@@ -349,7 +349,7 @@ The value of `chart` must be either a file in the `helm` directory of your Resou
 
 A value must be set to give the release a name and you may optionally set the namespace to use (otherwise the default on the deployment location is used). 
 
-You may also include the `values` option with the name of a file (also found in the helm directory) to customise the installation values. This file will be rendered as a template so you may inject Resource and/or system properties before it is used. 
```

Comment:
Do we mean to get rid of this line? (or is it just displaying strangely?)

Reply:
oops, removed this by accident, I've re-added with my latest commit